tour/beerDEcoded.yaml

```diff
@@ -104,7 +86,8 @@ steps:
       file'</b>: Yes<br> &nbsp;&nbsp;&nbsp;&nbsp;<b>'Format report output like
       Kraken 1’s kraken-mpa-report'</b>: Yes<br> <b>'Select a Kraken2
       database'</b>: fungi2019-03' <br><br>Click <b>'Next'</b> and the tour will
-      <b>'Execute'</b> the Kraken2 tool for you.'
+      <b>'Execute'</b> the Kraken2 tool for you.'<br><br><br>      
+      <small>Additional info:<br><b>Parameter ‘Single or paired reads’</b><br>Single-end reads are the fragments sequenced from one side. With paired-end sequencing, the fragments are sequenced from both sides. This approach results in two reads per fragment, with the first read in forward orientation and the second read in reverse-complement orientation. With this technique, we have the advantage to get more information about each DNA fragment compared to reads sequenced by only single-end sequencing<br><b>Parameter ‘Confidence'</b><br>A confidence score of 0.0 means that non-restrictive taxonomic assignation is desired. This value can be increased if a more restrictive taxonomic assignation is desired. For example, a confidence score of 0.1 means that at least 10% of the k-mers should match entries in the database. <br><b>Parameter ‘Select a Kraken2 database’</b><br>We need to identify the taxon to which the individual reads belong. To identify the origin of each sequence, taxonomic classification tools use  microbial genome databases. For this tutorial, we will use the <i>fungi2019-03</i> database.<small/><br>
```

Comment:
I would move Additional info block to the previous step ("Have a look at the tool's parameters of the Kraken2..")